functions/wav_time.h

```diff
@@ -0,0 +1,143 @@
+#ifndef FUNCTIONS_WAV_TIME_H
+#define FUNCTIONS_WAV_TIME_H
+
+#include "wav_effect.h"
+#include "int.h"
+#include "../blades/blade_base.h"
+
+// Usage: WavLen<EFFECT_NAME>  
+// EFFECT_NAMENUM : any off the available effect form the style edittor
+// Returns length of wav file returned as const int in [ms]
+// returned value: INTEGER
+//
+template<BladeEffectType EFFECT>
+class WavLen {
+ public:
+  void run(BladeBase* blade) {
+    found_ = false;
+    eff_name_ = wav_effect.GetEffectName(EFFECT);//find correspondig SoundEffectName for BladeEffectType
```

Comment:
why are we doing this by name? Why not by enum?

Reply:
We need to have the str comparison of the wav_player attribute